src/bedrock/monitoring.cpp

```diff
@@ -622,7 +638,7 @@ int main(int argc, char* argv[]) {
   // keep track of the keys' replication info
   unordered_map<string, key_info> placement;
   // warm up for benchmark
-  warmup(placement);
+  //warmup(placement);
```

Comment:
What does this do on the monitoring side? We should probably isolate benchmark specific code at some point.

Reply:
this warms up the benchmark keys' rep factors. But I notice that if we are not benchmarking, then the placement[key] won't have any info about the rep factor of a key, and this could be problematic... I will open an issue about this (when the placement doesn't contain info about a key, the monitoring thread should first query the rep factor from the server).